README.md

```diff
@@ -325,6 +325,21 @@ amadeus.travel.predictions.tripPurpose.get({
     departureDate: '2020-08-01',
     returnDate: '2020-08-12'
 })
+
+// Flight Delay Prediction
```

Comment:
@akshitsingla The dates are past ones. You can check the Python SDK has an updated real flight.